### PR TITLE
Updated labler + templates to match new scenario-based areas

### PR DESCRIPTION
**Pull request by kpathakota**

### CODE REVIEW

File: .github/config/issue-labler/labler-v1.yml [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-  - '- \[ ?(?:x|X) ?\] Workbench'
-area.publishing:
-  - '- \[ ?(?:x|X) ?\] Publishing'
-area.pods:
-  - '- \[ ?(?:x|X) ?\] Pods'
-area.markdown:
-  - '- \[ ?(?:x|X) ?\] Markdown'
-area.seeds:
-  - '- \[ ?(?:x|X) ?\] Seed'
+area.onboard:
```
Comment (flammehawk) [inline .github/config/issue-labler/labler-v1.yml:1]: You should not edit this file. 
Copy the content into `labler-v2.yml`

**Review by flammehawk — CHANGES_REQUESTED** [review]
Since we don't want to invalidate the already existing issues with the new labler, 
you need to create the wanted Config inside a `labler-v2.yml`

And you need to change the corresponding Variable in the Templates. 
This block `issue_labeler_regex_version=1`will need to have the 1 changed to a two in each template.

File: .github/config/issue-labler/labler-v1.yml (hunk 1 above)
Comment (kpathakota) [reply .github/config/issue-labler/labler-v1.yml:1]: ah, good to know, thanks.

**Comment by kpathakota** [comment]
@flammehawk - hope the updated changes work now. Haven't figured out a way to test other than merging it in ... so would be good to know if you've got any tips there.

**Review by flammehawk — APPROVED** [review]
Looks all good to me.

**Comment by flammehawk** [comment]
> @flammehawk - hope the updated changes work now. Haven't figured out a way to test other than merging it in ... so would be good to know if you've got any tips there.

The best bet to test would probably be to fork merge the changes and test it by creating an issue.

**Pull request merged**